app/models/article.rb

```diff
@@ -1,3 +1,28 @@
 class Article < ApplicationRecord
   belongs_to :category
+  has_one_attached :image
+
```

Comment:
why do we just have 1 image, an article can have many images

Reply:
Maybe we add this as an extra feature in the PT